logstash-core/lib/logstash/runner.rb

```diff
@@ -148,25 +147,29 @@ def run(args)
     begin
       LogStash::SETTINGS.from_yaml(LogStash::SETTINGS.get("path.settings"))
     rescue => e
-      @logger.subscribe(STDOUT)
-      @logger.warn("Logstash has a new settings file which defines start up time settings. This file is typically located in $LS_HOME/config or /etc/logstash. If you installed Logstash through a package and are starting it manually please specify the location to this settings file by passing in \"--path.settings=/path/..\" in the command line options")
-      @logger.fatal("Failed to load settings file from \"path.settings\". Aborting...", "path.settings" => LogStash::SETTINGS.get("path.settings"), "exception" => e.class, "message" => e.message)
+      logger.warn("Logstash has a new settings file which defines start up time settings. This file is typically located in $LS_HOME/config or /etc/logstash. If you installed Logstash through a package and are starting it manually please specify the location to this settings file by passing in \"--path.settings=/path/..\" in the command line options")
+      logger.fatal("Failed to load settings file from \"path.settings\". Aborting...", "path.settings" => LogStash::SETTINGS.get("path.settings"), "exception" => e.class, "message" => e.message)
```

Comment:
Currently, this is how this branch gets printed to the user. I am not necessarily annoyed by the first message about log4j, unless you think it should not be happening. I don't mind either way

```
ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console.
15:38:06.083 [main] FATAL logstash.runner - Failed to load settings file from "path.settings". Aborting... {"path.settings"=>"joids", "exception"=>Errno::ENOENT, "message"=>"No such file or directory - /Users/tal/Code/es/talevy-logstash/joids/logstash.yml"}
```